db/migrate/20170620222146_change_games_columnlast_move_pawn_two_steps_second_time.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeGamesColumnlastMovePawnTwoStepsSecondTime < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :games, :last_move_en_passant_situation?, :piece_capturable_by_en_passant
+    change_column :games, :piece_capturable_by_en_passant, :string
```

Comment:
If you set it up as an assocation, and make the column end in `_id`. Rails will store in the DB the ID, but when you call game.piece_capturable_by_en_passant. Rails will return the object.